### PR TITLE
Hide skeletons in the segmentation layer by default

### PR DESCRIPTION
**Pull request by tingzhao**
This PR is mainly for turning off skeleton display by default in the 'Annotate' workspace. While implementing this function, I refactored the layer creation part to remove duplicate code. This PR also includes a minor fix for canceling scheduled annotation table update when the annotation panel is unmounted.

### CODE REVIEW

**Review by neomorphic — DISMISSED** [review]
Looks good to me

File: src/Annotation/AnnotationPanel.jsx [hunk 1]
```diff
@@ -35,6 +35,8 @@ export default function AnnotationPanel(props) {
     const index = parseInt(newValue, 10);
     if (config.layers[index]) {
       actions.selectViewerLayer(config.layers[index].name);
+    } else {
```
Comment (neomorphic) [inline src/Annotation/AnnotationPanel.jsx:38]: Why call the else statement. Does something bad happen if we call nothing at all when the if condition is not met?

File: src/utils/neuroglancer.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    return 'segmentation';
+  }
+  if (layer.location.includes('segmentation')) {
+    return 'segmentation';
+  }
+  return undefined;
+}
+
+/* eslint-disable-next-line  import/prefer-default-export */
+export function addLayersFromDataset(layers, dataset, inferringType) {
```
Comment (neomorphic) [inline src/utils/neuroglancer.js:13]: This function has side effects, which can be confusing. How about changing it to

const updateLayers = addLayersFromDataset(layers, dataset, inferringType);

Then we know explicitly that the layers have been updated by this function.

**Review by neomorphic — COMMENTED** [review]
There are a few changes that I think should be made to the code before merging. Happy to talk about it.

File: src/Annotation/AnnotationPanel.jsx (hunk 1 above)
Comment (tingzhao) [reply src/Annotation/AnnotationPanel.jsx:38]: I added it on purpose to disable annotation layer selection when switching to merge/connection tab. If the annotation layer were kept on, the user might be confused by seeing an annotation added while [ctrl]clicking on the view accidentally.

File: src/utils/neuroglancer.js (hunk 2 above)
Comment (tingzhao) [reply src/utils/neuroglancer.js:13]: This looks good to me. I'll change it to remove side effects.

**Pull request merged**